### ISSUE TITLE
Aggregation issue on Mongojs 1.0.2 

### ISSUE THREAD

**Issue by mlecoq** [mafintosh/mongojs#218]
Hi,

I was using Mongojs 0.17. After upgrade to 1.0.2, my program was failing on agrégation with the following error message :  
pipeline element 0 is not an object

So I rolled back to version 0.18.2

Thanks,
Mickael.

**Comment by saintedlama** [comment]
Can you post some example code how you invoke the aggregation?

**Comment by mlecoq** [comment]
Sure !

Thanks for your reply.

My docs have the following fields: 
{
	"id" : "111111",
	"review" : {
		"text" : "blabla"
	},
	"_id" : ObjectId("545454")
}

 mongoDb.review.aggregate([{"$group" : {"_id" : {"review" : "$review.text" }, uniqueIds : {$push : {mongoId: "$_id", contributionId : "$id"}}, count: {$sum:1}  }}, {"$match" : {count : {"$gt" : 1} } } ], function (err, results){
     ...
  }


Mickael.

**Comment by saintedlama** [comment]
Don't pass the aggregation pipeline as an array but as arguments. That's a regression from 0.x versions to 1.0.x which was fixed by #204 which is not released yet.

Instead of 
```
mongoDb.review.aggregate([
  {"$group" : {"_id" : {"review" : "$review.text" }, uniqueIds : {$push : {mongoId: "$_id", contributionId : "$id"}}, count: {$sum:1} }}, 
  {"$match" : {count : {"$gt" : 1} } } 
], function (err, results){
```
Use
```
mongoDb.review.aggregate(
  {"$group" : {"_id" : {"review" : "$review.text" }, uniqueIds : {$push : {mongoId: "$_id", contributionId : "$id"}}, count: {$sum:1} }}, 
  {"$match" : {count : {"$gt" : 1} } }, 
  function (err, results){
```

Please check section __Upgrading from 0.x.x to 1.0.x__  in https://github.com/mafintosh/mongojs#upgrading-from-0xx-to-10x for more detail when migrating.

Thanks for reporting this issue will add another point to Upgrading from 0.x.x to 1.0.x section :+1:

**Comment by mlecoq** [comment]
Great. Thanks !
